src/assignment/assignment.helper.tsx

```diff
@@ -146,17 +144,17 @@ const VALIDATION_RULES_FOR_PUBLISH: ValidationRule<Partial<Avo.Assignment.Assign
 	{
 		error: tText('assignment/assignment___de-opdracht-bevat-geen-onderwijsniveaus'),
 		isValid: (assignment: Partial<Avo.Assignment.Assignment>) =>
-			validateLoms(assignment?.loms, LomType.educationDegree),
+			!!assignment.loms?.find((lom) => lom.lom?.scheme === LomSchemeType.structure),
```

Comment:
i wonder if it is worth it to use the groupLoms function for these as well? but then you need to check to arrays here.
your choice

Reply:
It might be easier this way because education degree and level are both options so it would take more effort to check 2 arrays of loms.